virtual_rainforest/models/animals/constants.py

```diff
@@ -0,0 +1,36 @@
+"""The `models.animals.constants` module contains a set of dataclasses containing
+constants" (fitting relationships taken from the literature) required by the broader
+:mod:`~virtual_rainforest.models.animals` module
+"""  # noqa: D205, D415
+
+from dataclasses import dataclass
+
+
+@dataclass
+class DamuthsLaw:
```

Comment:
Rather than having individual `dataclass`es for each of these constants with the constants hardcoded, I'd define one `dataclass` (maybe called `Term` or something...?) and make three separate instances of it, with the appropriate `exponent` and `coefficient`.